employee-rostering-frontend/src/ui/components/DataTable.test.tsx

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { DataTable, DataTableProps } from './DataTable';
+import { EditableComponent } from './EditableComponent';
+
+type MockData = [string, number];
+class MockDataTable extends DataTable<MockData, MockData, DataTableProps<MockData,MockData>> {
+  displayDataRow = jest.fn((data) => [<span key={0} id="viewer">data[0]</span>,
```

Comment:
`data[0]` is a text, not the first element of the `data` array. Is this on purpose or is it a mistake?

Same for `data[1]` on the next line.